server/jvm/{{appName}}-eventhandler/build.gradle.kts

```diff
@@ -1,13 +1,12 @@
 dependencies {
-    implementation("global.genesis:genesis-pal-execution")
-    implementation("global.genesis:genesis-eventhandler")
+    implementation(genesis("genesis-pal-execution"))
+    implementation(genesis("genesis-eventhandler"))
     implementation(project(":{{appName}}-messages"))
-    api("global.genesis:genesis-db")
+    api(genesis("genesis-db"))
     compileOnly(project(":{{appName}}-config"))
-    compileOnly(project(path = ":{{appName}}-dictionary-cache", configuration = "codeGen"))
-    testImplementation("global.genesis:genesis-dbtest")
-    testImplementation("global.genesis:genesis-testsupport")
-    testImplementation(project(path = ":{{appName}}-dictionary-cache", configuration = "codeGen"))
+    testImplementation(genesis("genesis-dbtest"))
+    testImplementation(genesis("genesis-testsupport"))
+    genesisGeneratedCode()
```

Comment:
I think here we will need to set the `withTestDependency` flag as the dictionary cache was also being added as a test dependency

Reply:
oh snap, didn't notice that arg, I thought they were both added by default

Reply:
No problem. There might be a case to enable it by default to be honest.